modules/cnf-du-partitioning-management-workloads.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-partitioning-management-workloads_{context}"]
+
+= Partitioning management workloads
+
+You can isolate the platform services, housekeeping tasks, and infrastructure pods to run on a reserved
+set of CPUs. This is useful for resource-constrained environments, such as single-node production deployments,
```

Comment:
I can't put my finger on the guideline, but can you replace "deployments" with "installations"?  I was asked to do the same.  I think it's fear-based that we not confuse an installation with a `Deployment` resource.

Reply:
@mikemckiernan The The whole concept of ZTP is deployment from a centralized hub cluster to single node clusters. I suppose it could be considered an installation, but deployment is much more accurate as we're deploying the same set of resources and policies to multiple clusters, albeit w/ different MAC addresses and IPs and such. You were also asked to change this?

Reply:
Sounds like maybe this is a `Deployment` resource.  As it was expressed to me, if `oc get deployments <deployment-name>` shows the name of the item in question, then it is definitely a `Deployment` resource and a general text description of "deployment" is OK.  If the term deployment refers to a cluster, then the preference is to avoid "deployment" in the text.

Reply:
I was unaware of this distiction. But note that we have existing documentation [Provisioning and deploying a distributed unit](https://docs.openshift.com/container-platform-ocp/4.7/scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.html) that uses the term extensively.

Reply:
After reading the content at the link, I'd say that's an installation rather than a deployment--not that civilization will crumble over the distinction.  I used the instance of "remote worker nodes" in the following sentence as my touchstone--"deployment" can be replaced with "installation" without loss of meaning--at least I think so.

```
This is done to address a deployment that includes remote worker nodes, with several sites belonging to the same cluster.
```